x-pack/plugins/cases/public/components/configure_cases/connectors_dropdown.tsx

```diff
@@ -15,6 +22,10 @@ import { useKibana } from '../../common/lib/kibana';
 import { getConnectorIcon, isDeprecatedConnector } from '../utils';
 import { euiStyled } from '../../../../../../src/plugins/kibana_react/common';
 
+type ConnectorEuiComboBoxOptionOption<T> = EuiComboBoxOptionOption<T> & {
```

Comment:
This custom type is to simplify the PR modifications and not move too much code around. EuiCombobox accepts a single function to render each item, superselect accept options with a rendered value. This is a middleway to make them compatible.